helm/syntho-ui/templates/backend/backend-service.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
 spec:
   ports:
     - name: backend-port
-      port: {{ .Values.backend.service.port }}
+      port: 80
```

Comment:
Previous frontend setup (the broken one) was trying to reach backend via a real domain. I changed it to in-cluster domain with previous port like `svc.namespace.blahblah:backendcontainerport`  then frontend was brokebn, Then I saw this and made it 80 in order access directly via in-cluster dns without providing explicit port. 

I'd say since this is related to service's exposed IP's 80th port which is like an encapsulated network for only this specific service, it is convenient to leave it like this. But if you have hard opinion on that, I could revert it.

Reply:
Yeah it's fine for now, I'll mark it as resolved!